src/BitcoinHelpers.js

```diff
@@ -698,39 +700,82 @@ const BitcoinHelpers = {
       expectedValue,
       outpoint
     ) {
-      const unspentTransactions = await electrumClient.getUnspentToScript(
+      const transactionsWithScript = await electrumClient.getTransactionsForScript(
```

Comment:
`transactionsForScript` would probably be a better term here to keep with the rest of the verbiage.